common/util.go

```diff
@@ -530,6 +530,12 @@ func CreateHistoryStartWorkflowRequest(
 		firstDecisionTaskBackoffSeconds += delayStartSeconds
 	}
 
+	// Add a random jitter to start time, if requested.
```

Comment:
Jitter should also apply to each occurrence of a workflow when there is a cron schedule. Otherwise, only the first run of a cron workflow will benefit from the jitter.

Reply:
I believe this is now working - see my latest commit.